depthmapXcli/linkparser.h

```diff
@@ -33,20 +43,39 @@ class LinkParser : public IModeParser
     virtual std::string getHelp() const
     {
         return  "Mode options for LINK:\n"\
-                "-lf <links file>\n"\
-                "-lnk <single link coordinates> provided in csv (x1,y1,x2,y2) for example \"0.1,0.2,0.2,0.4\" "\
-                "to create a link from 0.1,0.2 to 0.2,0.4. Provide multiple times for multiple links\n";
-
+                "  -lmt <type> Map type group to select displayed map from. One of:\n"\
+                "       pointmaps (default, vga: link)\n"\
+                "       shapegraphs (axial:link/unlink, segment:link, convex:link)\n"\
+                "  -lm  <mode> one of:\n"\
+                "       link (default)\n"\
+                "       unlink\n"\
+                "  -lt  <type> one of:\n"\
+                "       coords (default, provided as x,y or x1,y1,x2,y2 coordinates)\n"\
+                "       refs (provided as the ids (Ref) of the shapes)\n"\
+                "  -lnk <single link/unlink coordinates> provided in csv (x1,y1,x2,y2)\n"\
+                "       for example \"0.1,0.2,0.2,0.4\" to create a link from 0.1,0.2\n"\
+                "       to 0.2,0.4. In the case of axial-map unlinks a single (x,y) set may\n"
+                "       be provided. In the case of refs provide the ids in csv (reffrom,refto)"
+                "       Provide multiple times for multiple links/unlinks\n"\
+                "  -lf  <links file> as in -lnk\n";
     }
 
 public:
     virtual void parse(int argc, char *argv[]);
     virtual void run(const CommandLineParser &clp, IPerformanceSink &perfWriter) const;
 
     //link options
-    const std::vector<Line> & getMergeLines() const { return _mergeLines; }
+    const std::string & getLinksFile() const { return _linksFile; }
+    const std::vector<std::string> & getManualLinks() const { return _manualLinks; }
+    const MapTypeGroup& getMapTypeGroup() const { return _mapTypeGroup; }
+    const LinkMode& getLinkMode() const { return _linkMode; }
+    const LinkType& getLinkType() const { return _linkType; }
 private:
-    std::vector<Line> _mergeLines;
+    std::string _linksFile;
+    std::vector<std::string> _manualLinks;
+    MapTypeGroup _mapTypeGroup;
```

Comment:
Er, members starting with underscore are bad style in C++ (it's technically allowed, but questionable as it is not far away from reserved expressions, e.g. anything starting with a double underscore). Can we keep the style consistent by having members starting with a little `m`, please?